src/App.css

```diff
@@ -1,51 +1,109 @@
 body {
   margin: 0;
+  background-color: rgba(0, 0, 0, 0.05);
+  font-family: 'Montserrat', sans-serif;
+}
+
+.header {
+  text-transform: uppercase;
+  margin: 8rem 0 0 2rem;
+  font-size: 2rem;
+  color: #888;
+  font-weight: bold;
+}
+
+button {
+  padding: 1rem;
+  background-color: #0290ff;
+  border-radius: 4px;
+  width: 10rem;
+  height: 3rem;
+  color: #fff;
+  font-family: Montserrat, sans-serif;
+  font-size: larger;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#root {
+  padding: 3rem 5rem;
+  height: max-content;
+}
+
+.app {
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: #fafafa;
+  border-radius: 4px;
+  padding: 0 0 4rem;
 }
 
 li,
 a {
   text-decoration: none;
-  font-size: 2rem;
+  text-transform: uppercase;
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
 }
 
 nav {
   display: flex;
-  justify-content: space-around;
+  justify-content: flex-start;
+  gap: 3rem;
   align-items: center;
-  padding: 0 5rem;
-  gap: 35rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  height: 6rem;
+  background-color: #fff;
```

Comment:
- [ ] Please make your nav match the original design  by adding the user icon as well 

Your design :point_down: 

![image](https://user-images.githubusercontent.com/75126481/162587167-03d6d96f-b4a1-414a-9bf6-ae8bd8f942d6.png)

The original design :point_down: 

![image](https://user-images.githubusercontent.com/75126481/162587199-e80393b5-045a-442d-abb0-54d7688973f2.png)